src/app/util/process-global-message.cpp

```diff
@@ -642,12 +641,12 @@ bool emAfProcessGlobalCommand(EmberAfClusterCommand * cmd)
         // Ok. This is the command that matters.
         if (zclCmd == ZCL_DISCOVER_COMMANDS_RECEIVED_COMMAND_ID)
         {
-            emberAfPutInt8uInResp(ZCL_DISCOVER_COMMANDS_RECEIVED_RESPONSE_COMMAND_ID);
+            emberAfPutInt32uInResp(ZCL_DISCOVER_COMMANDS_RECEIVED_RESPONSE_COMMAND_ID);
             flag = false;
         }
         else
         {
-            emberAfPutInt8uInResp(ZCL_DISCOVER_COMMANDS_GENERATED_RESPONSE_COMMAND_ID);
+            emberAfPutInt32uInResp(ZCL_DISCOVER_COMMANDS_GENERATED_RESPONSE_COMMAND_ID);
```

Comment:
I really wish we had added an emberAfPutCommandIdInResp and same for reading and for other types...  but since we plan to switch to TLV from this stuff anyway, probably not worth it.